clang/tools/driver/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set( LLVM_LINK_COMPONENTS
   ScalarOpts
   Support
   SYCLLowerIR
+  SYCLTransforms
```

Comment:
Are there any changes in FE compiler/driver source code? If so, there should be no need to link this library.

Reply:
this is legacy code from high-perf host device